### ISSUE TITLE
Create pt-br version 🇧🇷

### ISSUE THREAD

**Issue by brunopadz** [brunopadz/aws-devops-exam-prep-guide#2]
Create a Portuguese Brazilian version of the guide.

**Comment by leandrocostam** [comment]
I can help you with! Let me know :)

**Comment by brunopadz** [comment]
> I can help you with! Let me know :)

Yeah, I'm thinking about the best way to do it since the new exam will be out sometime next month.
